Gruntfile.js

```diff
@@ -336,7 +336,8 @@ module.exports = function(grunt) {
 					[ WORKING_DIR + 'wp-includes/js/wp-util.js' ]: [ './src/js/_enqueues/wp/util.js' ],
 					[ WORKING_DIR + 'wp-includes/js/wpdialog.js' ]: [ './src/js/_enqueues/lib/dialog.js' ],
 					[ WORKING_DIR + 'wp-includes/js/wplink.js' ]: [ './src/js/_enqueues/lib/link.js' ],
-					[ WORKING_DIR + 'wp-includes/js/zxcvbn-async.js' ]: [ './src/js/_enqueues/lib/zxcvbn-async.js' ]
+					[ WORKING_DIR + 'wp-includes/js/zxcvbn-async.js' ]: [ './src/js/_enqueues/lib/zxcvbn-async.js' ],
+					[ WORKING_DIR + 'wp-includes/js/webp-fallback.js' ]: [ './src/js/_enqueues/wp/webp-fallback.js' ]
```

Comment:
These files are alphabetically sorted, so we should put this into the right place in the list.

Also, while this is right now focused on WebP, it could be expanded to other MIME types in the future. How about we name the file `image-mime-fallback.js`, or even better, `wp-image-mime-fallback.js` (since most WP owned scripts have the WP prefix)?